core/src/main/java/org/apache/carbondata/core/util/CarbonUtil.java

```diff
@@ -1536,7 +1537,8 @@ public static ValueEncoderMeta deserializeEncoderMetaV2(byte[] encoderMeta) {
     ValueEncoderMeta meta = null;
     try {
       aos = new ByteArrayInputStream(encoderMeta);
-      objStream = new ObjectInputStream(aos);
+      objStream =
```

Comment:
"CarbonUtil.closeStreams(objStream);" cann't be called when not IOException

Reply:
no need for other exception handling because stream throw only ClassNotFoundException/IOException.